docs/server-request.md

```diff
@@ -21,14 +21,38 @@ A low-level action that allows for any kind of HTTP request to be performed.
 
 - [Parse JSON](/docs/Actions/CallApi.md) - creates variables from JSON returned by API calls
 - [Regex](/docs/Actions/CallApi.md) - create variables by matching patterns inside the content
+- [Execute Actions Async](/docs/Actions/ExecuteAsync.md) - nesting other actions under the Execute Actions Async enables them to continue executing without waiting for the previous actions to finalize. Useful in the case of Server Request action when you don't want to wait for it to bring a response.
 
-## `Parameter Reference`
+## `Input Parameter Reference`
 
 | Parameter     | Description                           | Supports Variables |
 |---------------|---------------------------------------|--------------------|
-| URL           | Absolute URL where the resource to be invoked lives. | Yes |
-| HTTP Method   | The operations to perform against the URL.<br/> Possible values are GET, POST, PUT, DELETE, HEAD. | No |
+| URL           | The URL where the resource to be invoked lives. It can be relative or absolute. | Yes |
+| URL Token Encoding | URLs brought from variables might contain characters outside of the ASCII character-set, this will pose problems when you are trying to send requests to those URLs. <br/>URL encoding takes care of this issue by converting unsafe characters to valid ASCII characters. For example, a whitespace, is not a ASCII character and it will be replaced with '%20' or the '+' sign. If you are using variables to provide the URL, the option URL Encoding will make sure the URL is valid. | No |
```

Comment:
I believe URL encoding is not soo much about encoding special characters as it's about encoding the characters that have meaning in a URL, such as /, &, =.